src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -49,11 +46,11 @@
     </AssemblyMetadata>
   </ItemGroup>
   <!-- Platform specific properties -->
-  <PropertyGroup Condition="'$(Platform)' == 'amd64'">
+  <PropertyGroup Condition="'$(Platform)' == 'x64'">
     <PlatformTarget>x64</PlatformTarget>
     <Prefer32Bit>false</Prefer32Bit>
     <BaseAddress>0x180000000</BaseAddress>
-    <DefineConstants>BIT64;AMD64;$(DefineConstants)</DefineConstants>
+    <DefineConstants>BIT64;$(DefineConstants)</DefineConstants>
```

Comment:
You cannot just stop defining `AMD64` here. It is used when compiling CoreLib in multiple places.

We want to keep using AMD64 define for x64. It is pretty entrenched for historic reasons - changing it would be uphill battle.

Reply:
Only $(Platform) should be changed, to make it consistent with rest of world.

Reply:
Why makes changes here at all as part of this change? Why not a separate, targeted change?